website/pages/en/index.js

```diff
@@ -277,6 +277,16 @@ const Hero = ({ language }) => (
       <p>
         <translate>Use next generation JavaScript, today.</translate>
       </p>
+
+      <div className="hero__announcement">
+        <span>
+          <strong>Babel 7 is out!</strong> Please read our{" "}
```

Comment:
Maybe we could put this on two lines

Reply:
![image](https://user-images.githubusercontent.com/56288/44678840-ef841c00-a9fe-11e8-853d-28f32a051359.png)

Think it looks fine?

Reply:
Yeah I think it's a good length